build-system/tasks/presubmit-checks.js

```diff
@@ -164,8 +164,8 @@ var forbiddenTerms = {
   'installStorageService': {
     message: privateServiceFactory,
     whitelist: [
-      'extensions/amp-analytics/0.1/amp-analytics.js',
-      'extensions/amp-analytics/0.1/storage-impl.js',
+      'src/runtime.js',
```

Comment:
Calling it from a JS binary where it it is not intended to be called,
bloats that JS size.

On Tue, Sep 13, 2016 at 12:52 PM, Mohammad Khatib <notifications@github.com>
wrote:

> In build-system/tasks/presubmit-checks.js
> <https://github.com/ampproject/amphtml/pull/4944#discussion_r78631462>:
>
> > @@ -164,8 +164,8 @@ var forbiddenTerms = {
> >    'installStorageService': {
> >      message: privateServiceFactory,
> >      whitelist: [
> > -      'extensions/amp-analytics/0.1/amp-analytics.js',
> > -      'extensions/amp-analytics/0.1/storage-impl.js',
> > +      'src/runtime.js',
>
> I am actually not sure, it seems we do still list a lot of installXService.
> So I think this will stay here. Not sure why exactly we don't allow calling
> this.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ampproject/amphtml/pull/4944/files/e7ef5f2855a6d613f4d144f00c12554ff589b69c#r78631462>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFeT79ZDvtDxz0fmed7ba9Gw714bD-Iks5qpv7zgaJpZM4J7GGg>
> .
>